.travis.yml

```
@@ -1,7 +1,8 @@
-
 jobs:
   include:
-    - name: Unit Tests
+    - stage: "Tests"
+      name: "Unit Tests"
+      if: type = pull_request OR (type = push && branch =~ /^master|[0-9]+-(dev|stable)$/) OR type = cron
```

nitpicking: unneeded parenthesis and mix of `||` / `&&` with `AND` / `OR`
```suggestion
      if: type = pull_request OR type = push AND branch =~ /^master|[0-9]+-(dev|stable)$/ OR type = cron
```

Plus: we don't have any `*-stable` branch: why include this pattern?

---

I include this pattern to avoid forgetting it